actions/actions.py

```diff
@@ -309,6 +303,24 @@ def submit(
             ]
             vendor = ""
 
+        time_range = tracker.get_slot("time")
+
+        if ("to" not in time_range) or ("from" not in time_range):
+            from_date = date.fromisoformat(time_range.split("T")[0])
+            to_date = date.today()
+        else:
+            from_date = date.fromisoformat(
+                time_range.get("from").split("T")[0]
+            )
+            to_date = date.fromisoformat(time_range.get("to").split("T")[0])
```

Comment:
This confused me a little, maybe it is easier to say
```
if "to" in time_range and "from" in time_range:

else:
```
(those are the 2 different cases, right? they would be flipped in that case ofc)

Reply:
Yep, makes sense. Conditions are flipped now